### ISSUE TITLE
ValueError on user-related query

### ISSUE THREAD

**Issue by foxbunny** [Outernet-Project/librarian-auth#1]
This exception is raised querying the database that was freshly created with today's code (@3802a6ef1). Not 100% sure if it's auth-related.

```
Traceback (most recent call last):
  File "/home/hajime/.envs/libtest/bin/bottle.py", line 862, in _handle
    return route.call(**args)
  File "/home/hajime/code/librarian-core/librarian_core/contrib/databases/plugins.py", line 11, in wrapper
    return callback(*args, **kwargs)
  File "/home/hajime/.envs/libtest/lib/python2.7/site-packages/bottle_utils/i18n.py", line 434, in wrapper
    return callback(*args, **kwargs)
  File "/home/hajime/code/librarian-core/librarian_core/contrib/sessions/plugins.py", line 37, in wrapper
    return callback(*args, **kwargs)
  File "/home/hajime/.envs/libtest/bin/bottle.py", line 1729, in wrapper
    rv = callback(*a, **ka)
  File "/home/hajime/code/librarian-auth/librarian_auth/plugins.py", line 33, in wrapper
    return callback(*args, **kwargs)
  File "/home/hajime/code/librarian-setup/librarian_setup/plugins.py", line 14, in wrapper
    if (not supervisor.exts.setup_wizard.is_completed and
  File "/home/hajime/code/librarian-setup/librarian_setup/setup.py", line 83, in is_completed
    self._is_completed = not self.get_needed_steps()
  File "/home/hajime/code/librarian-setup/librarian_setup/wizard.py", line 251, in get_needed_steps
    return [idx for idx, step in self.steps.items()
  File "/home/hajime/code/librarian-core/librarian_core/contrib/databases/squery.py", line 33, in from_utc_timestamp
    dt = datetime.datetime.utcfromtimestamp(float(timestamp))
ValueError: invalid literal for float(): 2015-10-16T17:23:11+00:00
```

**Comment by foxbunny** [comment]
Recreating the database again seems to have made it go away. Keeping it open in case it's an intermittent issue.

**Comment by foxbunny** [comment]
Nope, didn't fix it. Manifests itself on next log-in.

**Comment by afaikafk** [comment]
Fixed in 4d0d49da7d953337528602ad75aa4377b70f8766 , issue was with deserializing the user object from the session where it was sitting in json encoded string
